### DISCUSSION TITLE
Fullscreen document editor or changable width

### DISCUSSION THREAD

**Discussion by EwanLyon** [keystonejs/keystone#8147] [Feature requests]
We recently started using the document editor to create a page for our event but some of our elements in the page are quite large and take up the entire width of the document editor. This makes it difficult to work with those components. I now understand that the component blocks used on keystone need to be specifically designed for the document editor but I still find the width to be limiting and almost claustrophobic.

Letting users increase the width or fullscreening the document editor would alleviate these issues.

Example. (Logo blurred cus I don't wanna advertising here lol)
![image](https://user-images.githubusercontent.com/19513845/205196417-f9a6e683-932a-477c-aaa2-95d084674e79.png)

**Comment by jordie23** [comment]
I would love to have a wider editor space.

I think the "expand" button on the document editor used to go to full screen but it has since been change to just expand to the full height of the content. Unsure of the reason sorry, may have been buggy in full screen.

I think the sidebar could be reduced in width to give the main content column more space. It's currently a lot of space for a menu.

I think that's all coming though, their roadmap suggests they're working on new admin UI: https://keystonejs.com/updates/roadmap

> A next-gen Admin UI that unifies developer and editor collaborations in new and exciting ways
